packages/ui/lib/Preview/index.js

```diff
@@ -16,10 +16,10 @@ const PreviewWrapper = styled.div`
     `}
 `;
 
-function Preview({ content, bordered = true, minHeight }) {
+function Preview({ content, bordered = true, allowTags, minHeight }) {
```

Comment:
can we add default values for `allowTags, minHeight`?

also, it will be great pratice to put props that has default value to the tail of the list

Reply:
can not for both, 

allowTags `[]` will render as plain text, meaning: lose markdown features.
minHeight is used in `RichEditor` only.

Reply:
its fine to define common tags into `allowTags` eg `p, a, img ...etc`， just refer real world usage
for `minHeight`, lets set a `19.25px` for now, which represents one single line